checkov/main.py

```diff
@@ -249,6 +251,14 @@ def run(banner: str = checkov_banner, argv: List[str] = sys.argv[1:]) -> Optiona
                                                         source_version=source_version,
                                                         repo_branch=config.branch,
                                                         prisma_api_url=config.prisma_api_url)
+
+            if source.name in [BCSourceType.GITHUB_ACTIONS, BCSourceType.JENKINS, BCSourceType.CIRCLECI, BCSourceType.CODEBUILD] \
```

Comment:
@maxamel Maybe we should add to the `BCSource` enum another field, `is_ci`, and have it by default `False` but for these 4 have it as `True`?

Reply:
Done